common.h

```diff
@@ -289,6 +289,7 @@ typedef struct {
     const char *origFileName;
     char fileName[PATH_MAX];
     char crashFileName[PATH_MAX];
+    char ext[PATH_MAX];
```

Comment:
That can be `char ext[NAME_MAX];` instead (for less size overhead) since the extension cannot be larger than the allowed max file name.